feathr-example/installment-payments_credit_card_balance.ipynb

```diff
@@ -0,0 +1,1073 @@
+    "  f_CNT_DRAWINGS_OTHER_CURRENT,\n",
+    "  f_CNT_DRAWINGS_POS_CURRENT,\n",
+    "  f_CNT_INSTALMENT_MATURE_CUM,\n",
+    "  f_NAME_CONTRACT_STATUS,\n",
+    "  f_SK_DPD,\n",
+    "  f_SK_DPD_DEF,\n",
+    "\n",
+    "  f_CREDIT_CARD_BALANCE_EMA_AVG\n",
+    "\n",
+    "  ]\n",
+    "\n",
+    "anchor_credit_card_balance_core = FeatureAnchor(name=\"anchor_credit_card_balance_core\",\n",
+    "                                source=credit_card_balance_source_core,\n",
+    "                                features=features_credit_card_balance_core)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "client.build_features(\n",
+    "    anchor_list=[\n",
+    "        anchor_installments_payments_core,\n",
+    "        anchor_credit_card_balance_core\n",
+    "        ], \n",
+    "    derived_feature_list=[])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create training data using point-in-time correct feature join\n",
+    "\n",
+    "A training dataset usually contains entity id columns, multiple feature columns, event timestamp column and label/target column. \n",
+    "\n",
+    "To create a training dataset using Feathr, one needs to provide a feature join configuration file to specify\n",
+    "what features and how these features should be joined to the observation data. The feature join config file mainly contains: \n",
+    "\n",
+    "1. The path of a dataset as the 'spine' for the to-be-created training dataset. We call this input 'spine' dataset the 'observation'\n",
+    "   dataset. Typically, each row of the observation data contains: \n",
+    "   a) Column(s) representing entity id(s), which will be used as the join key to look up(join) feature value. \n",
+    "   b) A column representing the event time of the row. By default, Feathr will make sure the feature values joined have\n",
+    "   a timestamp earlier than it, ensuring no data leakage in the resulting training dataset. \n",
+    "   c) Other columns will be simply pass through onto the output training dataset.\n",
+    "2. The key fields from the observation data, which are used to joined with the feature data.\n",
+    "3. List of feature names to be joined with the observation data. The features must be defined in the feature\n",
+    "   definition configs.\n",
+    "4. The time information of the observation data used to compare with the feature's timestamp during the join.\n",
+    "\n",
+    "Create training dataset via:\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2022-07-07 09:53:13.989 | INFO     | feathr._synapse_submission:upload_or_get_cloud_path:62 - Uploading /var/folders/gs/dbrzk90d0m3849n982_q27w40000gn/T/tmp8pnag696/feathr_pyspark_driver.py to cloud..\n",
```

Comment:
Logs

Reply:
Cleaned up the outputs of the notebooks.